source/app/service-providers/assets/types.config-provider.d.ts

```diff
@@ -145,6 +145,7 @@ interface ConfigOptions {
     deleteOnFail: boolean
     leaveAppRunning: boolean
     avoidNewTabs: boolean
+    leaveAppRunning: boolean
```

Comment:
`leaveAppRunning` is already defined two lines above. An error when merging? I suggest don't change `types.config-provider.d.ts`.